CONTRIBUTING.md

```diff
@@ -354,6 +354,22 @@ You can rebuild iD completely with the command `npm run all`.
 iD will be built to the `dist` directory. This directory is self-contained; you can copy it
 into the public directory of your webserver to deploy iD.
 
+### Build Troubleshooting
+
+**`npm install` fails with "PhantomJS not found on PATH … Try to install
+PhantomJS globally".**
+Then you should do exactly that, such as `apt install phantomjs`.
```

Comment:
If https://github.com/openstreetmap/id#prerequisites is outdated, it should be updated